### PR TITLE
Update community guidelines

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Update and rename content strategy document
- Add user documentation guidelines
- Update documentation toggle / tabs docu
- Update Preview docu

**Related issue(s)**

### CODE REVIEW

File: docs/guidelines/content-guidelines/05-tabs-toggles.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+</div>
+```
+
+The code renders on `kyma-project.io` as follows:
+
+![](./assets/toggle-flat-structure.png)
+
+* See the example of the toggle added under a list:
+
+```html
```
Comment (NHingerl) [inline docs/guidelines/content-guidelines/05-tabs-toggles.md:159]: all lines of the code block need 2 more blank spaces to render as part of the list item

```suggestion
  ```html
```

Screenshot 2023-09-04 at 14 00 22

File: docs/guidelines/content-guidelines/10-docs-preview.md [hunk 2]
```diff
@@ -1,51 +1,26 @@
...
 Before you publish new documentation on the Kyma website, you can preview your changes to see if the formatting of the text is correct, images fit well, and links work as expected.
 
-That is possible thanks to the **preview feature** supported by [Netlify](https://www.netlify.com/). It attaches links to autogenerated previews of all website views to your pull requests (PRs).
+That is possible thanks to the [**docsify preview feature**](https://docsify.js.org/#/quickstart?id=preview-your-site) supported by [docsify](https://docsify.js.org/#/).
 
-This feature is enabled on these Kyma repositories:
+## Prerequisites
 
-- [`kyma`](https://github.com/kyma-project/kyma) for changes in the `/docs` folder that contains sources of the official Kyma documentation rendered in [kyma-project.io](https://kyma-project.io).
+Docsify preview, requires docsify-cli.
```
Comment (NHingerl) [inline docs/guidelines/content-guidelines/10-docs-preview.md:9]:
```suggestion
You have installed docsify-cli.
```

**Review by NHingerl — COMMENTED** [review]
mostly rendering issues that are probably fixed with indentation

File: docs/guidelines/content-guidelines/01-user-docs.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+2. Add a new `_sidebar.md` file including a link to your document, or update an existing `_siderbar.md` to include it.
+
+## Publish a document from an existing module repositiory
+
+To publish a document located in an existing module repositiory, follow these steps:
+
+1. Create a pull request adding your content to a Markdown file located in the `/docs/user` folder in your module repository.
+2. Add a new `_sidebar.md` file including a link to your document, or update an existing `_siderbar.md` to include it.
+
+> **CAUTION** When you create or update `_sidebar.md` files, bear in mind that link to your document may be added in multiple `_sidebar.md` files.
```
[the remark this answers is not in the document]
Comment (NHingerl) [reply docs/guidelines/content-guidelines/01-user-docs.md]: Has been rephrased and is clear now :)

File: docs/guidelines/content-guidelines/01-user-docs.md [hunk 4]
```diff
@@ -22,9 +22,30 @@ To publish a document located in the `/kyma` repository, follow these steps:
...
-To publish a document located in an existing module repositiory, follow these steps:
+To publish a document located in an existing module repositiory, follow these steps.
 
 1. Create a pull request adding your content to a Markdown file located in the `/docs/user` folder in your module repository.
-2. Add a new `_sidebar.md` file including a link to your document, or update an existing `_siderbar.md` to include it.
+2. In the `docs/user` folder, add a new `_sidebar.md` file including a link to your document, or update the existing `_siderbar.md` to include it.
+
+## Publish a document from a new module repostory
+
+To publish a document located in a new module repositiory, follow the steps from [Publish a document from an existing module](#publish-a-document-from-an-existing-module-repositiory). Once completed do the following:
```
Comment (NHingerl) [inline docs/guidelines/content-guidelines/01-user-docs.md:32]:
```suggestion
To publish a document located in a new module repositiory, follow the steps from [Publish a document from an existing module](#publish-a-document-from-an-existing-module-repositiory). Once completed, do the following:
```

File: docs/guidelines/content-guidelines/01-user-docs.md [hunk 5]
```diff
@@ -22,9 +22,30 @@ To publish a document located in the `/kyma` repository, follow these steps:
...
+    For example:
+
+    ```html
+    alias: {
+    '/serverless-manager/(.*)': 'https://raw.githubusercontent.com/kyma-project/serverless-manager/main/docs/$1',
+    }, 
+    ```
 
-> **CAUTION** When you create or update `_sidebar.md` files, bear in mind that link to your document may be added in multiple `_sidebar.md` files.
+> **CAUTION** When you update navigation paths in documentaion, make sure you check all `_sidebar.md` files that may be affected.
```
Comment (NHingerl) [inline docs/guidelines/content-guidelines/01-user-docs.md:51]:
```suggestion
> **CAUTION** When you update navigation paths in documentation, make sure you check all `_sidebar.md` files that may be affected.
```

File: docs/guidelines/content-guidelines/01-user-docs.md [hunk 6]
```diff
@@ -22,9 +22,30 @@ To publish a document located in the `/kyma` repository, follow these steps:
...
 
 1. Create a pull request adding your content to a Markdown file located in the `/docs/user` folder in your module repository.
-2. Add a new `_sidebar.md` file including a link to your document, or update an existing `_siderbar.md` to include it.
+2. In the `docs/user` folder, add a new `_sidebar.md` file including a link to your document, or update the existing `_siderbar.md` to include it.
+
+## Publish a document from a new module repostory
+
+To publish a document located in a new module repositiory, follow the steps from [Publish a document from an existing module](#publish-a-document-from-an-existing-module-repositiory). Once completed do the following:
+
+1. In the `/kyma` repository, add a link to your module documentation main page under **Modules** in the [`/kyma/docs/_sidebar.md`](https://github.com/kyma-project/kyma/blob/main/docs/_sidebar.md) file.
```
Comment (NHingerl) [inline docs/guidelines/content-guidelines/01-user-docs.md:34]:
```suggestion
1. In the `/kyma` repository, in the [`/kyma/docs/_sidebar.md`](https://github.com/kyma-project/kyma/blob/main/docs/_sidebar.md) file, under **Modules**, add a link to your module documentation main page.
```

I'd specify the exact location first and then the instruction. However, it sounds a bit inelegant, so maybe rather:

```suggestion
1. In the `/kyma` repository, open the [`/kyma/docs/_sidebar.md`](https://github.com/kyma-project/kyma/blob/main/docs/_sidebar.md) file and under **Modules**, add a link to your module documentation main page.
```

File: docs/guidelines/content-guidelines/05-tabs-toggles.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+
+For example, you can use tabs and/or documentation toggle to include instructions for different operating systems (OS) in one guide, or to describe installation guides that differ only in a few steps.
+
+As described in the document, tabs are a docsify-specific form of presenting content versions in Markdown. Toggles use HTML within a Markdown document.
+
+## Docsify tabs
+
+To display tabbed content from Markdown, `kyma-project.io` uses the [docsify-tabs plugin](https://jhildenbiddle.github.io/docsify-tabs/#/) from [docsify.js](https://docsify.js.org/#/).
+
+You define a tab set using `tabs:start` and `tabs:end` HTML comments. For detailed guidelines, see the [Usage section in the plugin official documentation](https://jhildenbiddle.github.io/docsify-tabs/#/?id=usage).
```
Comment (NHingerl) [inline docs/guidelines/content-guidelines/05-tabs-toggles.md:15]:
```suggestion
You define a tab set using `tabs:start` and `tabs:end` HTML comments. For detailed guidelines, see the [Usage section in the plugin's official documentation](https://jhildenbiddle.github.io/docsify-tabs/#/?id=usage).
```
or, shorter:

```suggestion
You define a tab set using `tabs:start` and `tabs:end` HTML comments. For detailed guidelines, see [docsify-tabs: Usage](https://jhildenbiddle.github.io/docsify-tabs/#/?id=usage).
```

File: docs/guidelines/content-guidelines/01-user-docs.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
+# User documentation
+
+Open-source Kyma and Kyma module user documentation is displayed on `https://kyma-project.io`. The website uses [docsify](https://docsify.js.org/#/) as a documentation site generator.
+
+The overaching Kyma content is pulled from the `/docs` folder in the `/kyma` repository. Module documentation is pulled from the `/docs/user` folder in respective module repositories.
```
Comment (grego952) [inline docs/guidelines/content-guidelines/01-user-docs.md:5]:
```suggestion
The overarching Kyma content is pulled from the `/docs` folder in the `/kyma` repository. Module documentation is pulled from the `/docs/user` folder in respective module repositories.
```

File: docs/guidelines/content-guidelines/01-user-docs.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+After initialization, to run, docsify uses the follwing files located in the `/docs` folder in the `/kyma` repository:
```
Comment (grego952) [inline docs/guidelines/content-guidelines/01-user-docs.md:7]:
```suggestion
After initialization, to run, docsify uses the following files located in the `/docs` folder in the `/kyma` repository:
```

File: docs/guidelines/content-guidelines/05-tabs-toggles.md [hunk 10]
```diff
@@ -0,0 +1,234 @@
...
+
+For example, you can use tabs and/or documentation toggle to include instructions for different operating systems (OS) in one guide, or to describe installation guides that differ only in a few steps.
+
+As described in the document, tabs are a docsify-specific form of presenting content versions in Markdown. Toggles use HTML within a Markdown document.
+
+## Docsify tabs
+
+To display tabbed content from Markdown, `kyma-project.io` uses the [docsify-tabs plugin](https://jhildenbiddle.github.io/docsify-tabs/#/) from [docsify.js](https://docsify.js.org/#/).
+
+You define a tab set using `tabs:start` and `tabs:end` HTML comments. For detailed guidelines, see [docsify-tabs: Usage](https://jhildenbiddle.github.io/docsify-tabs/#/?id=usage).
```
Comment (grego952) [inline docs/guidelines/content-guidelines/05-tabs-toggles.md:15]:
```suggestion
You define a tab set using the `tabs:start` and `tabs:end` HTML comments. For detailed guidelines, see [docsify-tabs: Usage](https://jhildenbiddle.github.io/docsify-tabs/#/?id=usage).
```

File: docs/guidelines/content-guidelines/05-tabs-toggles.md [hunk 11]
```diff
@@ -0,0 +1,234 @@
...
+
+  - No spaces for the div element
+  - Two spaces for other elements
+
+* Use the following indentation when you add a toggle under a list:
+
+  - Four spaces for the div element
+  - Six spaces for other elements
+
+* When you add two or more toggle sets that belong to the same group, you must classify them properly. This way each, time you switch to one of the toggles in the set, it opens automatically in all other sets from the same group within one document and across all documents that use the group. To create a toggle group:
```
Comment (grego952) [inline docs/guidelines/content-guidelines/05-tabs-toggles.md:61]:
```suggestion
* When you add two or more toggle sets that belong to the same group, you must classify them properly. This way, each time you switch to one of the toggles in the set, it opens automatically in all other sets from the same group within one document and across all documents that use the group. To create a toggle group:
```

File: docs/guidelines/content-guidelines/06-links.md [hunk 12]
```diff
@@ -28,7 +28,7 @@ To add a reference to a YAML, JSON, SVG, PNG, or JPG file located in the `assets
 
 ## Links in documentation toggles
 
-To link to a document in a documentation toggle, the toggle must start with the `<div tabs name="{toggle-name}">` tag and end with the `</div>` tag, where **name** is a distinctive ID used for linking. For more information, read a separate [document on toggles](./05-documentation-toggle.md).
+To link to a document in a documentation toggle, the toggle must start with the `<div tabs name="{toggle-name}">` tag and end with the `</div>` tag, where **name** is a distinctive ID used for linking. For more information, read a separate [Tabs & toggles](./05-tabs.md) document.
```
Comment (NHingerl) [inline docs/guidelines/content-guidelines/06-links.md:31]: or even shorter:

```suggestion
To link to a document in a documentation toggle, the toggle must start with the `<div tabs name="{toggle-name}">` tag and end with the `</div>` tag, where **name** is a distinctive ID used for linking. For more information, read [Tabs & toggles](./05-tabs.md).
```

File: docs/guidelines/content-guidelines/10-docs-preview.md [hunk 13]
```diff
@@ -1,51 +1,26 @@
...
-
-3. Copy the [`build-preview.sh`](https://github.com/kyma-project/cli/blob/main/.kyma-project-io/build-preview.sh) script to the `.kyma-project-io` folder in the new repository. In this script, change the name of the repository under the exported **APP_PREPARE_FOR_REPO** environment:
-
-    ```bash
-    export APP_PREPARE_FOR_REPO="{REPOSITORY_NAME}"
-    ```
-
-4. Merge changes and [Netlify](https://www.netlify.com/) will start generating website previews on PRs in the new repository.
+1. In your module repository, create a pull request with documentation changes.
+2. In the `/kyma` repository, go to the [`docs/index.html`](https://github.com/kyma-project/kyma/blob/main/docs/index.html) file and change the value of the **alias** parameter for your module. By default the value points to the raw version of the `/docs/user` folder on the main branch of your module repository. Change the value to point to the raw version of the respective folder in your pull request or on your branch.
```
Comment (nataliasitko) [inline docs/guidelines/content-guidelines/10-docs-preview.md:23]:
```suggestion
2. In the `/kyma` repository, go to the [`docs/index.html`](https://github.com/kyma-project/kyma/blob/main/docs/index.html) file and change the value of the **alias** parameter for your module. By default, the value points to the raw version of the `/docs` folder on the main branch of your module repository. Change the value to point to the raw version of the respective folder on your branch.
```
I'd say all aliases are set to point to the `/docs` folder in our module repos. If you want to preview changes from your pull request, you must point to the branch and path in your forked repo that contains the changes:

```html
alias: {
'/{MODULE_REPOSITORY_NAME}/(.*)': 'https://raw.githubusercontent.com/{YOUR_GITHUB_USERNAME}/{FORKED_MODULE_REPOSITORY_NAME}/{BRANCH_TO_PREVIEW}/docs/$1',
}, 
```
For example:
```html
alias: {
'/istio/(.*)': 'https://raw.githubusercontent.com/kymauser/istio/docu-update/docs/$1',
}, 
```
I'm not sure if you can also point to the pull request

**Review by grego952 — APPROVED** [review]

**Pull request merged**